src/latexify/codegen/function_codegen.py

```diff
@@ -536,6 +536,23 @@ def visit_IfExp(self, node: ast.IfExp) -> str:
         latex += self.visit(node)
         return latex + r", & \mathrm{otherwise} \end{array} \right."
 
+    def visit_Match(self, node: ast.Match) -> str:
+        """Visit a match node"""
+        latex = r"\left\{ \begin{array}{ll} "
+        subject_latex = self.visit(node.subject)        
+        for i, match_case in enumerate(node.cases):
+            true_latex = self.visit(match_case.body[0])
```

Comment:
we need to check if `match_case.body` has exactly one statement, or raise an appropriate exception in `latexify.exceptions`.
we also need to add a test case to capture this workflow.